changelog/0043-optionally-configure-auditcare-db.yml

```diff
@@ -0,0 +1,42 @@
+title: Optionally configure auditcare db
+key: optionally_configure_auditcare_db
+date: 2021-03-23
+optional_per_env: yes
+min_commcare_version:
+max_commcare_version:
+context: |
+  Some enterprise deployments of CommCare use a backend feature called "auditcare" as part of
+  their audit logging strategy. Historically it has used CouchDB as its data backend,
+  but it is being switched to use PostgreSQL instead. If you care about this feature then
+  you may want to carefully consider this change log before your next commcare deploy; otherwise you can ignore.
+
+details: |
+  After the next deploy from the latest version of CommCare HQ, auditcare will begin writing to PostgreSQL.
+  By default, it will write to tables within your default PostgreSQL db.
+  If you want to have it write to a different database instead, you can configure this using the
+  `LOCAL_CUSTOM_DB_ROUTING` variable and an entry in postgresql.yml.
+update_steps: |
+  There are no special steps if you are fine with the default.
+
+  To configure auditcare to go to its own PostgreSQL database instead, use the following:
+  ```
+  # public.yml
+  LOCAL_CUSTOM_DB_ROUTING:
+    auditcare: auditcare
+  ```
+
+  If you have more than one postgres machine and you want auditcare on one other than the default one,
+  update your postgresql.yml to point to the right machine. If you have only one postgres machine
+  or want to use your default postgres machine, you can skip this step.
```

Comment:
I think an `auditcare:` entry is required in postgresql.yml if you want to have auditcare write to a separate database, so you cannot "skip this step." If this step is skipped the auditcare database will not be created.

Reply:
For other dbs you don't have to. If you don't mention a db, it'll default to pointing to `DEFAULT_POSTGRESQL_HOST` which defaults to the first `postgresql` host. For example you can see in https://github.com/dimagi/commcare-cloud/blob/e6b6b8f0c1cec257c520c3ac2fe11b4f749088f8/environments/swiss/postgresql.yml#L23-L26 that not much is specified under `dbs`.

Reply:
Daniel and I offlined, and https://github.com/dimagi/commcare-cloud/blob/31e36f990a9714bd81c3579973b93473a776a16e/src/commcare_cloud/environment/schemas/postgresql.py#L301 is what I missing.